build.sh

```diff
@@ -34,11 +34,15 @@ SUDO=""
 
 PRINT_HELP=0
 
-FEATURES="CAMERA COLORD FLICKR GRAPHICSMAGICK IMAGEMAGICK KWALLET LIBSECRET LUA MAP MAC_INTEGRATION NLS OPENCL OPENEXR OPENMP UNITY WEBP GAME"
+FEATURES="USE_CAMERA_SUPPORT USE_COLORD USE_FLICKR USE_GRAPHICSMAGICK USE_IMAGEMAGICK "
+FEATURES+="USE_KWALLET USE_LIBSECRET USE_LUA USE_MAP USE_MAC_INTEGRATION USE_NLS USE_OPENCL USE_OPENEXR "
+FEATURES+="USE_OPENMP USE_UNITY USE_WEBP USE_OPENJPEG USE_GAME USE_GMIC USE_LENSFUN USE_XMLLINT USE_ICU "
```

Comment:
I won't expose GAME anymore as we don't want it to be exposed and prefer to have it discouraged.

Reply:
So then parsing `DefineOptions.cmake` to keep build options in sync with the build script is also not encouraged I guess :sweat_smile: